app/controllers/home_controller.rb

```diff
@@ -1,21 +1,21 @@
 class HomeController < ApplicationController
   def index
-    @client = IEX::Api::Client.new(
-      publishable_token: Rails.application.credentials.config[:aws][:publishable_token],
-      secret_token: Rails.application.credentials.config[:aws][:secret_token],
-      endpoint: 'https://sandbox.iexapis.com/stable'
-    )
+    #   # @client = IEX::Api::Client.new(
+    #   #   publishable_token: Rails.application.credentials.config[:aws][:publishable_token],
+    #   #   secret_token: Rails.application.credentials.config[:aws][:secret_token],
+    #   #   endpoint: 'https://sandbox.iexapis.com/stable'
+    #   # )
 
-    @crypto = @client.crypto('BTCUSDT')
-    @bitcoin = @crypto.company_name
-    @active = @client.stock_market_list(:mostactive)
-    @sectors = @client.sectors('MARKET')
-    # @all = client.crypto('SQL')
-    # @market = @client.market
-    # @spy = client.market['SPY'].close.price
-    @fb = @client.price('FB')
-    # @fb_logo = client.logo('FB')
-    @chart = @client.chart('MSFT', 'id', chart_simplify: true)
-    @logo = @client.logo('MSFT')
+    #   # @crypto = @client.crypto('BTCUSDT')
+    #   # @bitcoin = @crypto.company_name
+    #   # @active = @client.stock_market_list(:mostactive)
+    #   # @sectors = @client.sectors('MARKET')
+    #   # # @all = client.crypto('SQL')
+    #   # # @market = @client.market
+    #   # # @spy = client.market['SPY'].close.price
+    #   # @fb = @client.price('FB')
+    #   # # @fb_logo = client.logo('FB')
+    #   # @chart = @client.chart('MSFT', 'id', chart_simplify: true)
+    #   # @logo = @client.logo('MSFT')
```

Comment:
> I commented out the API as of the moment since it hinders the localhost from loading when it's uncommented.

I think you have to delete this code and figure out how to improve this code's performance

Reply:
Okay maam. Got this. Will delete the code first.